Tests/InfomaniakCoreTests/Chunking/TestRangeProviderConfig.swift

```diff
@@ -0,0 +1,30 @@
+/*
+ Infomaniak kDrive - iOS App
+ Copyright (C) 2023 Infomaniak Network SA
+
+ This program is free software: you can redistribute it and/or modify
+ it under the terms of the GNU General Public License as published by
+ the Free Software Foundation, either version 3 of the License, or
+ (at your option) any later version.
+
+ This program is distributed in the hope that it will be useful,
+ but WITHOUT ANY WARRANTY; without even the implied warranty of
+ MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ GNU General Public License for more details.
+
+ You should have received a copy of the GNU General Public License
+ along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+@testable import InfomaniakCore
+
+enum TestRangeProviderConfig {
+    static let `default` = RangeProvider.Config(
```

Comment:
Is it not more "kDrive" than default ? Could we keep it outside of the tests ?

Also maybe you could add the configuration for ST along kDrive.

Reply:
My point of view is that Core shouldn't be aware that other apps exist. 

Each app should provide it's own config. We shouldn't have to modify Core each time we want to modify a value for the apps.

Maybe the name in the tests is bit confusing. The goal of `TestRangeProviderConfig` is to provide a/some config for tests.

Reply:
Yeah I think it has value to be able to test with the actual config of both our apps.
I do see the issue about decoupling, yet this is our core lib so that's less of an issue, as it can provide more value.

Reply:
Sure let's go on this way 👍